### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jweakley**
Simplified the documentation.

### CODE REVIEW

**Comment by chadzink** [comment]
Nice update, this was confusing when I was using the tool last week. Much simpler now.
:+1:

File: README.md [hunk 1]
```diff
@@ -10,29 +10,8 @@ Add the gem to your Gemfile:
...
-To add or create a user with token authentication run:
-`rails generate token_authenticate_me:install <model>`
-
-Replace `<model>` with the class name used for users. This will create the necessary migration files, and optionally create the model file if it does not exist.
-
-**Right now this gem only supports creating the authentication model `User`, so it is recommended to call `rails generate token_authenticate_me:install user`**
-
-Include TokenAuthenticateMe::TokenAuthentication into the application controller or any controllers that require authorization:
-````rb
-require 'token_authenticate_me/controllers/token_authenticateable'
```
Comment (Samsinite) [inline README.md]: This snippet is still correct except for the path to the controller concern, which is now `'token_authenticate_me/concenrs/controllers/token_authenticateable' and `TokenAuthenticateMe::Concerns::Controllers:TokenAuthenticateable`

Comment (Samsinite) [inline README.md]: Suggest keeping this section just fixing the invalid parts as it is still needed for controller authentication.

File: README.md [hunk 2]
```diff
@@ -10,19 +10,15 @@ Add the gem to your Gemfile:
...
-`rails generate token_authenticate_me:install <model>`
+To install run the following:
+`rails generate token_authenticate_me:install`
 
-Replace `<model>` with the class name used for users. This will create the necessary migration files, and optionally create the model file if it does not exist.
-
-**Right now this gem only supports creating the authentication model `User`, so it is recommended to call `rails generate token_authenticate_me:install user`**
-
-Include TokenAuthenticateMe::TokenAuthentication into the application controller or any controllers that require authorization:
+Include TokenAuthenticateMe::TokenAuthenticateable into the application controller or any controllers that require authorization:
```
Comment (Samsinite) [inline README.md]: I believe the correct namespace is `TokenAuthenticateMe::Concerns::Controllers::TokenAuthenticateable`

**Pull request merged**